.github/workflows/build.yml

```diff
@@ -133,6 +133,9 @@ jobs:
       matrix:
         arch: [arm]
         toolset: ['14.2', '14.1'] # VS 2019, 2017
+        include:
+        - arch: arm # No arm64 prompt exists
+          toolset: '14.0' # VS 2015
```

Comment:
What is the purpose of this?
Why comment that arm64 does not exist, do you intend to add every single unsupported architecture as comment?

Reply:
> What is the purpose of this?

To actually support VS2015, obviously.
Comment moved to Jira.